Sources/Data/models/SessionTab.swift

```diff
@@ -176,12 +179,14 @@ extension SessionTab {
   
   public static func updateAll(tabs: [(tabId: UUID, interactionState: Data, title: String, url: URL)]) {
     DataController.performOnMainContext { context in
-      for (index, tab) in tabs.enumerated() {
-        guard let sessionTab = Self.from(tabId: tab.tabId, in: context) else { return }
+      for tab in tabs {
+        guard let sessionTab = Self.from(tabId: tab.tabId, in: context) else {
+          Logger.module.error("Error: SessionTab.updateAll missing managed object")
+          continue
+        }
         sessionTab.interactionState = tab.interactionState
         sessionTab.title = tab.title
         sessionTab.url = tab.url
-        sessionTab.index = Int32(index)
```

Comment:
Is this change intended?

Reply:
Yeah it is intended so tabs keep their index.